_posts/2023-03-16-making-easier-to-contribute.md

```diff
@@ -0,0 +1,80 @@
+
+  [subtest that occasionally times out]
+    expected: [PASS, TIMEOUT]
+```
+
+In the future, it may be worth migrating the existing intermittent issues to expectations like this.
+
+## Retrying tests with unexpected results
+
+Sometimes the causes of flakiness can affect many or even all tests, like bugs causing some reftest screenshots to be completely white, or overloaded test runners causing some tests to time out.
+
+Thanks to [#29370](https://github.com/servo/servo/pull/29370), we now retry tests that yield unexpected results. If a test yields the expected result on the second try, we ignore it when deciding whether or not the build should succeed. This can make builds a little slower, but it should be outweighed by our recent improvements to build times.
+
+In the future, it may be worth adopting some more advanced retry techniques. For example, [Chromium’s retry strategy](https://chromium.googlesource.com/chromium/src/+/9539073eb08b163ffa375bec280e7043ee4cbb19/docs/infra/cq.md) includes retrying the entire “shard” of tests to reproduce the test environment more accurately, and retrying tests both with and without the pull request to help “exonerate” the changes. These techniques require considerably more resources though, and they are generally only viable if we can [fund our own dedicated test runners](https://servo.org/sponsorship/).
+
+## Result comments
+
+As of [#29315](https://github.com/servo/servo/pull/29315), when a try or merge build finishes, we now post a comment on the pull request with a clear breakdown of the unexpected results:
+
+* **Flaky unexpected results** are those that were unexpected at first, but expected on retry
+* **Stable unexpected results that are known to be intermittent** are those that were unexpected, but ignored due to being [marked as intermittent](https://github.com/servo/servo/issues?q=label%3AI-intermittent)
+* **Stable unexpected results** are those that caused the build to fail
+
+## Intermittent dashboard
+
+To ensure that flaky tests can be discovered and fixed even if they are mitigated by retries, we’ve created an [intermittent dashboard](https://build.servo.org/intermittent-tracker/) that all unexpected results get reported to.
+
+Each result includes the test and subtest, the expected and actual outcomes, any test output, plus metadata like the commit and a link to the build. You can filter the data to a specific test or field value, and the dashboard automatically points out when all of the visible results have something in common, which can help us analyse the failures and identify patterns.
+
+For example, here we can see that all of the unexpected failures for one of the HTML parsing tests have the same assertion failure on the same subtest, but are not limited to one pull request:
+
+![screenshot of intermittent dashboard, filtered by test (/html/syntax/parsing/DOMContentLoaded-defer.html) and actual outcome (FAIL)]({{ site.url }}/img/blog/intermittent-dashboard.png)
```

Comment:
```suggestion
![screenshot of intermittent dashboard, filtered by test (/html/syntax/parsing/DOMContentLoaded-defer.html) and actual outcome (FAIL)]({{ '/img/blog/intermittent-dashboard.png' | url }})
```

This should fix the image in the preview.

Reply:
done in b8922d0e96a97d56105b54b797cb086a884fed0a